snapshot/lib/snapshot.rb

```diff
@@ -17,3 +17,46 @@
 require_relative 'snapshot/simulator_launchers/simulator_launcher'
 require_relative 'snapshot/simulator_launchers/simulator_launcher_xcode_8'
 require_relative 'snapshot/module'
+
+require 'fastlane_core'
+
+require 'open3'
+
+module Snapshot
```

Comment:
When you rebase onto current `master`, this module will alredy exist in `snapshot/module` relative from here.

Reply:
If not, and this is just a useless copy, one would have to make sure if there are no important changes here.

Reply:
Quick comparison shows that they are indeed identical. This is probably just an artifact (that has to be fixed before a merge).